### ISSUE TITLE
security/acme-client: can't update txt record for Hurricane Electric

### ISSUE THREAD

**Issue by fpn** [opnsense/plugins#334]
```
[Mon Oct 23 15:48:55 PDT 2017] d_api='/usr/local/share/examples/acme.sh/dnsapi/dns_he.sh'
[Mon Oct 23 15:48:55 PDT 2017] Found domain api file: /usr/local/share/examples/acme.sh/dnsapi/dns_he.sh
[Mon Oct 23 15:48:55 PDT 2017] Using DNS-01 Hurricane Electric hook
[Mon Oct 23 15:48:55 PDT 2017] No auth details provided. Please set user credentials using the $HE_Username and $HE_Password envoronment variables.
[Mon Oct 23 15:48:55 PDT 2017] Error add txt for domain:_acme-challenge....
```

Also the username / password is empty the next time I open the "Validation Method":

![image](https://user-images.githubusercontent.com/1007715/31917586-202c823c-b80d-11e7-8d3f-fc1ec2a82314.png)

**Comment by fraenki** [comment]
@fpn Thanks for your report! I'll look into it.

**Comment by fraenki** [comment]
@fpn If you're running the latest version you can test the fix like this:

```
opnsense-patch -c plugins 1acb8c8
```

You'll have to enter your HE username and password again, unfortunately.

**Comment by fpn** [comment]
Thank you ! I have to admit that I just went in and edited dns_he.sh to make it work ;-)

**Comment by fraenki** [comment]
The fix will be in plugin version 1.12 (possibly next week).
